src/python/test/test_nextflow.py

```diff
@@ -203,7 +203,7 @@ def test_dx_build_nextflow_from_repository_default_metadata(self):
         pipeline_name = "hello"
         hello_repo_url = "https://github.com/nextflow-io/hello"
         applet_id = run(
-            "dx build --nextflow --repository '{}' --brief".format(hello_repo_url)).strip()
+            "dx build --nextflow --repository '{}' --brief | jq .id -r".format(hello_repo_url)).strip()
```

Comment:
Thanks for the fix. This LGTM.

A sidenote for the future: you can also use [json.loads](https://github.com/dnanexus/dx-toolkit/blob/ea014fdf22bf970b800286cfba16f693fcf28f06/src/python/test/test_nextflow.py#L175-L176) (in case `jq` is not available in the test environment).

Reply:
I've changed it and rerun the tests.